core/src/main/scala/org/bitcoins/core/channels/Channel.scala

```diff
@@ -209,85 +251,125 @@ sealed trait ChannelInProgress extends Channel with BaseInProgress {
   }
 
   /** Updates the payment channel with the given parameters */
-  private def updateChannel(outPoint: TransactionOutPoint, creditingTx: Transaction,
-    outputs: Seq[TransactionOutput],
-    clientKey: ECPrivateKey)(implicit ec: ExecutionContext): Future[ChannelInProgressClientSigned] = {
-    val partiallySigned = EscrowTimeoutHelper.clientSign(outPoint, creditingTx, outputs,
-      clientKey, lock, HashType.sigHashSingleAnyoneCanPay)
-    val inProgress = partiallySigned.map(tx =>
-      ChannelInProgressClientSigned(anchorTx, lock, clientChangeSPK, tx, current +: old))
+  private def updateChannel(outPoint: TransactionOutPoint,
+                            creditingTx: Transaction,
+                            outputs: Seq[TransactionOutput],
+                            clientKey: ECPrivateKey)(
+      implicit ec: ExecutionContext): Future[ChannelInProgressClientSigned] = {
+    val partiallySigned = EscrowTimeoutHelper.clientSign(
+      outPoint,
+      creditingTx,
+      outputs,
+      clientKey,
+      lock,
+      HashType.sigHashSingleAnyoneCanPay)
+    val inProgress = partiallySigned.map(
+      tx =>
```

Comment:
This seems like a weird format

Reply:
As it stands, lambda parameters that come inside curly braces will not have a new line but those inside parentheses will as there isn't currently an option in scalafmt to change this